lib/bpk-component-map/stories.js

```diff
@@ -115,6 +121,97 @@ const StatefulPriceMarkersExample = () => {
   );
 };
 
+const StatefulMapMarkersExample = () => {
+  const [selectedVenue, setSelectedVenue] = useState('2');
+
+  const venues = [
+    {
+      id: '1',
+      name: 'Glencoe Museum',
+      latitude: 35.68,
+      longitude: 139.69,
+      price: '£48',
+      disabled: false,
+      icon: icons.landmark,
+    },
+    {
+      id: '2',
+      name: 'Abisko Inn & Suites',
+      latitude: 35.67,
+      longitude: 139.7,
+      price: '£151',
+      disabled: false,
+      icon: icons.landmark,
+    },
+    {
+      id: '3',
+      name: 'The Panjin Lounge',
+      latitude: 35.65,
+      longitude: 139.71,
+      price: '£62',
+      disabled: false,
+      icon: icons.landmark,
+    },
+    {
+      id: '4',
+      name: 'Nara Bed & Breakfast',
+      latitude: 35.63,
+      longitude: 139.7,
+      price: '£342',
+      disabled: false,
+      icon: icons.landmark,
+    },
+    {
+      id: '5',
+      name: 'Kolkata Springs Hotel',
+      latitude: 35.635,
+      longitude: 139.72,
+      price: 'Sold out',
+      disabled: true,
+      icon: icons.landmark,
```

Comment:
This is totally not a big deal so I don't mind at all if you push back, but it might be more fun if we used some different icons here to give some variety and renamed them from hotels, e.g. "FC Nara Stadium", "Panjin Aquarium", you get the idea

Reply:
Sure thing!

Reply:
Made some new naming changes and icons to match!